### PR TITLE
Added MCDOC

### PR DESCRIPTION
**Pull request by XtronXI**
https://github.com/bs-community/awesome-minecraft/issues/35

**What is MCDOC?**

> MCDOC is the ultimate list for all Minecraft related Tools and Unlockers. Its contains tools for Bedrock, Education, China, Legends, Dungeons, Earth and their cross platform versions but doesnt contain Java (since there are already enough tools in awesome-minecraft and others). It also wraps up the Story of Cracking MCBE, Secret Marketplace Items and Guides.

**Website:** [https://mcdoc.openm.tech](https://mcdoc.openm.tech/)

> ***Published by [OpenM Team](https://github.com/OpenM-Project)***

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -50,6 +50,10 @@
 - [Minecraft Forums](https://www.minecraftforum.net/) - Official community forums, a great place to talk about the game.
 - [Planet Minecraft](https://www.planetminecraft.com/forums/) - Creative community fansite.
 
+### Tools
```
Comment (g-plane) [inline readme.md:53]: I think it should be "Documentations", not "Tools"?

Comment (XtronXI) [reply readme.md:53]: Hmm idk, it feels not kinda it for it. Maybe "Indexes"?
if not "Indexes" then its Documentation

**Review by XtronXI — COMMENTED** [review]
Is this okey?

**Review by g-plane — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (ars-fr) [reply readme.md]: yeah, tools isn't right, either documentation or indexes is good.

**Pull request merged**